tensorflow_serving/model_servers/main.cc

```diff
@@ -70,6 +70,7 @@ limitations under the License.
 #include "tensorflow_serving/apis/prediction_service.pb.h"
 #include "tensorflow_serving/config/model_server_config.pb.h"
 #include "tensorflow_serving/core/availability_preserving_policy.h"
+#include "tensorflow_serving/model_servers/config_reloader.h"
```

Comment:
You're going to have to edit the BUILD file. I don't understand how it can work otherwise to add the new source files. Maybe you forgot to incorporate the BUILD file changes into the PR?